docs/reference/ml/apis/put-dfanalytics.asciidoc

```diff
@@ -0,0 +1,105 @@
+[role="xpack"]
+[testenv="platinum"]
+[[put-dfanalytics]]
+=== Create {dfanalytics-jobs} API
+[subs="attributes"]
+++++
+<titleabbrev>Create {dfanalytics-jobs}</titleabbrev>
+++++
+
+beta[]
+
+Instantiates a {dfanalytics-job}.
+
+[[ml-put-dfanalytics-request]]
+==== {api-request-title}
+
+`PUT _ml/data_frame/analytics/<data_frame_analytics_id>`
+
+[[ml-put-dfanalytics-prereq]]
+==== {api-prereq-title}
+
+* You must have `machine_learning_admin` built-in role to use this API. You must 
+also have `read` and `view_index_metadata` privileges on the source index and 
+`read`, `create_index`, and `index` privileges on the destination index. For 
+more information, see {stack-ov}/security-privileges.html[Security privileges] 
+and {stack-ov}/built-in-roles.html[Built-in roles].
+
+[[ml-put-dfanalytics-path-params]]
+==== {api-path-parms-title}
+
+`<data_frame_analytics_id>` (Required)::
+  (string) A numerical character string that uniquely identifies the 
+  {dfanalytics-job}. This identifier can contain lowercase alphanumeric characters 
+  (a-z and 0-9), hyphens, and underscores. It must start and end with alphanumeric 
+  characters.
+
+[[ml-put-dfanalytics-request-body]]
+==== {api-request-body-title}
+
+`source` (Required)::
+  (object) The source configuration, consisting of `index` and optionally a 
```

Comment:
Just making a note it will be nice to add links to the resource page here where we have explicit sections discussing the structure of these objects.